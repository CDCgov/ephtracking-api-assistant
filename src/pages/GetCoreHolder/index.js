import React, { Component, Fragment } from 'react';

import CodeBlock from '../../components/CodeBlock';
import CIM from '../../components/CIM';
import GeographicFilter from '../../components/GeographicFilter'
import Preview from '../../components/Preview';
import SelectGeographicType from '../../containers/SelectGeographicType';
import SelectStratificationLevel from '../../containers/SelectStratificationLevel';
import SelectYears from '../../containers/SelectYears';

class GetCoreHolder extends Component {
  constructor(props) {
    super(props);
    this.state = {
      measureId: null,
      geographicTypeId: null,
      stratificationLevelId: null,
      geographicTypeIdFilter: 'ALL',
      geographicItemsFilter: 'ALL',
      years: [],
      isSmoothed: false,
      getFullCoreHolder: false,
      queryParams: ''
    };
    this.setMeasureId = this.setMeasureId.bind(this);
    this.setGeographicTypeId = this.setGeographicTypeId.bind(this);
    this.setStratificationLevelId = this.setStratificationLevelId.bind(this);
    this.handleInputChange = this.handleInputChange.bind(this);
    this.setYears = this.setYears.bind(this);
    this.setGeographicFilter = this.setGeographicFilter.bind(this);
  }
  
  setMeasureId(measureId) {
    this.setState({
      measureId,
      geographicTypeId: null,
      stratificationLevelId: null,
      years: [],
      geographicTypeIdFilter: 'ALL',
      geographicItemsFilter: 'ALL',
     });
  }

  setGeographicTypeId(geographicTypeId) {
    this.setState({
      geographicTypeId,
      stratificationLevelId: null,
      geographicTypeIdFilter: 'ALL',
      geographicItemsFilter: 'ALL',
    });
  }

  setStratificationLevelId(stratificationLevelId, queryParams) {
    this.setState({ stratificationLevelId, queryParams });
  }

  setYears(years) {
    this.setState({ years });
  }

  setGeographicFilter(filter) {
    this.setState({
      geographicTypeIdFilter: filter.geographicTypeIdFilter,
      geographicItemsFilter: filter.geographicItemsFilter
    });
  }

  handleInputChange(event) {
    const target = event.target;
    const value = target.type === 'checkbox' ? target.checked : target.value;
    const name = target.name;
    this.setState({
      [name]: value
    });
  }

  render() {
    const { measureId, geographicTypeId, stratificationLevelId, geographicTypeIdFilter, geographicItemsFilter, years, isSmoothed, getFullCoreHolder, queryParams } = this.state;
    const isValidUrl = measureId !== null && stratificationLevelId !== null
      && geographicTypeIdFilter !== null && geographicItemsFilter !== null && years.length > 0;
    const url = `https://ephtracking.cdc.gov/apigateway/api/v1/getCoreHolder/${measureId}/${stratificationLevelId}/${geographicTypeIdFilter}/${geographicItemsFilter}/${years.toString()}/${isSmoothed ? 1 : 0}/${getFullCoreHolder ? 1 : 0}${queryParams.length > 0 ? `?${queryParams}` : ''}`

    return (
      <Fragment>
        <h1 className="title">Retrieving all Values for a Measure</h1>
        <h5 className="title is-5">Usage</h5>
        <CodeBlock>
          https://ephtracking.cdc.gov/apigateway/api/{'{'}version{'}'}/getCoreHolder/{'{'}measureId{'}'}/{'{'}stratificationLevelId{'}'}/{'{'}geographicTypeIdFilter{'}'}/{'{'}geographicItemsFilter{'}'}/{'{'}temporal{'}'}/{'{'}isSmoothed{'}'}/{'{'}getFullCoreHolder{'}'}[?apiToken][?Variables...]
        </CodeBlock>
        <hr />
        <h5 className="title is-5">Set parameters</h5>
        <CIM handleSelect={this.setMeasureId} />
        <SelectGeographicType measureId={measureId} handleSelect={this.setGeographicTypeId} />
        <SelectStratificationLevel measureId={measureId} geographicTypeId={geographicTypeId} handleSelect={this.setStratificationLevelId} />
        <GeographicFilter measureId={measureId} geographicTypeId={geographicTypeId} handleSelect={this.setGeographicFilter} />
        <SelectYears measureId={measureId} handleCheck={this.setYears} />
        <div className="field">
          <label className="checkbox">
            <input name="isSmoothed" type="checkbox" checked={this.state.isSmoothed} onChange={this.handleInputChange} className="input-checkbox" />
            Is Smoothed
          </label>
          <p className="help">Note: the majority of measures do not have smoothing values</p>
        </div>
        <div className="field">
          <label className="checkbox">
            <input name="getFullCoreHolder" type="checkbox" checked={this.state.getFullCoreHolder} onChange={this.handleInputChange} className="input-checkbox" />
            getFullCoreHolder
          </label>
        </div>
        <hr />
        <Preview url={url} validUrl={isValidUrl} />
      </Fragment>
    );
  }
}

export default GetCoreHolder;
