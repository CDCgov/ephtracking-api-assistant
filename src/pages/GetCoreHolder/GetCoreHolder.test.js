import React from 'react';
import { shallow } from 'enzyme';

import GetCoreHolder from './';

describe('Page GetCoreHolder', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GetCoreHolder />);
    expect(wrapper).toMatchSnapshot();
  });
});
