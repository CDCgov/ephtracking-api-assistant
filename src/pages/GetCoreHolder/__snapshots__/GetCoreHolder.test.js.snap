// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page GetCoreHolder renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GetCoreHolder />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1
          className="title"
>
          Retrieving all Values for a Measure
</h1>,
        <h5
          className="title is-5"
>
          Usage
</h5>,
        <CodeBlock>
          https://ephtracking.cdc.gov/apigateway/api/
          {
          version
          }
          /getCoreHolder/
          {
          measureId
          }
          /
          {
          stratificationLevelId
          }
          /
          {
          geographicTypeIdFilter
          }
          /
          {
          geographicItemsFilter
          }
          /
          {
          temporal
          }
          /
          {
          isSmoothed
          }
          /
          {
          getFullCoreHolder
          }
          [?apiToken][?Variables...]
</CodeBlock>,
        <hr />,
        <h5
          className="title is-5"
>
          Set parameters
</h5>,
        <CIM
          handleSelect={[Function]}
/>,
        <SelectGeographicType
          handleSelect={[Function]}
          measureId={null}
/>,
        <SelectStratificationLevel
          geographicTypeId={null}
          handleSelect={[Function]}
          measureId={null}
/>,
        <GeographicFilter
          geographicTypeId={null}
          handleSelect={[Function]}
          measureId={null}
/>,
        <SelectYears
          handleCheck={[Function]}
          measureId={null}
/>,
        <div
          className="field"
>
          <label
                    className="checkbox"
          >
                    <input
                              checked={false}
                              className="input-checkbox"
                              name="isSmoothed"
                              onChange={[Function]}
                              type="checkbox"
                    />
                    Is Smoothed
          </label>
          <p
                    className="help"
          >
                    Note: the majority of measures do not have smoothing values
          </p>
</div>,
        <div
          className="field"
>
          <label
                    className="checkbox"
          >
                    <input
                              checked={false}
                              className="input-checkbox"
                              name="getFullCoreHolder"
                              onChange={[Function]}
                              type="checkbox"
                    />
                    getFullCoreHolder
          </label>
</div>,
        <hr />,
        <Preview
          url="https://ephtracking.cdc.gov/apigateway/api/v1/getCoreHolder/null/null/ALL/ALL//0/0"
          validUrl={false}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Retrieving all Values for a Measure",
          "className": "title",
        },
        "ref": null,
        "rendered": "Retrieving all Values for a Measure",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Usage",
          "className": "title is-5",
        },
        "ref": null,
        "rendered": "Usage",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            "https://ephtracking.cdc.gov/apigateway/api/",
            "{",
            "version",
            "}",
            "/getCoreHolder/",
            "{",
            "measureId",
            "}",
            "/",
            "{",
            "stratificationLevelId",
            "}",
            "/",
            "{",
            "geographicTypeIdFilter",
            "}",
            "/",
            "{",
            "geographicItemsFilter",
            "}",
            "/",
            "{",
            "temporal",
            "}",
            "/",
            "{",
            "isSmoothed",
            "}",
            "/",
            "{",
            "getFullCoreHolder",
            "}",
            "[?apiToken][?Variables...]",
          ],
        },
        "ref": null,
        "rendered": Array [
          "https://ephtracking.cdc.gov/apigateway/api/",
          "{",
          "version",
          "}",
          "/getCoreHolder/",
          "{",
          "measureId",
          "}",
          "/",
          "{",
          "stratificationLevelId",
          "}",
          "/",
          "{",
          "geographicTypeIdFilter",
          "}",
          "/",
          "{",
          "geographicItemsFilter",
          "}",
          "/",
          "{",
          "temporal",
          "}",
          "/",
          "{",
          "isSmoothed",
          "}",
          "/",
          "{",
          "getFullCoreHolder",
          "}",
          "[?apiToken][?Variables...]",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Set parameters",
          "className": "title is-5",
        },
        "ref": null,
        "rendered": "Set parameters",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "handleSelect": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "handleSelect": [Function],
          "measureId": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "geographicTypeId": null,
          "handleSelect": [Function],
          "measureId": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "geographicTypeId": null,
          "handleSelect": [Function],
          "measureId": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "handleCheck": [Function],
          "measureId": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <label
              className="checkbox"
>
              <input
                            checked={false}
                            className="input-checkbox"
                            name="isSmoothed"
                            onChange={[Function]}
                            type="checkbox"
              />
              Is Smoothed
</label>,
            <p
              className="help"
>
              Note: the majority of measures do not have smoothing values
</p>,
          ],
          "className": "field",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <input
                  checked={false}
                  className="input-checkbox"
                  name="isSmoothed"
                  onChange={[Function]}
                  type="checkbox"
/>,
                "Is Smoothed",
              ],
              "className": "checkbox",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "checked": false,
                  "className": "input-checkbox",
                  "name": "isSmoothed",
                  "onChange": [Function],
                  "type": "checkbox",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
              "Is Smoothed",
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Note: the majority of measures do not have smoothing values",
              "className": "help",
            },
            "ref": null,
            "rendered": "Note: the majority of measures do not have smoothing values",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <label
            className="checkbox"
>
            <input
                        checked={false}
                        className="input-checkbox"
                        name="getFullCoreHolder"
                        onChange={[Function]}
                        type="checkbox"
            />
            getFullCoreHolder
</label>,
          "className": "field",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                checked={false}
                className="input-checkbox"
                name="getFullCoreHolder"
                onChange={[Function]}
                type="checkbox"
/>,
              "getFullCoreHolder",
            ],
            "className": "checkbox",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "checked": false,
                "className": "input-checkbox",
                "name": "getFullCoreHolder",
                "onChange": [Function],
                "type": "checkbox",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            "getFullCoreHolder",
          ],
          "type": "label",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "url": "https://ephtracking.cdc.gov/apigateway/api/v1/getCoreHolder/null/null/ALL/ALL//0/0",
          "validUrl": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1
            className="title"
>
            Retrieving all Values for a Measure
</h1>,
          <h5
            className="title is-5"
>
            Usage
</h5>,
          <CodeBlock>
            https://ephtracking.cdc.gov/apigateway/api/
            {
            version
            }
            /getCoreHolder/
            {
            measureId
            }
            /
            {
            stratificationLevelId
            }
            /
            {
            geographicTypeIdFilter
            }
            /
            {
            geographicItemsFilter
            }
            /
            {
            temporal
            }
            /
            {
            isSmoothed
            }
            /
            {
            getFullCoreHolder
            }
            [?apiToken][?Variables...]
</CodeBlock>,
          <hr />,
          <h5
            className="title is-5"
>
            Set parameters
</h5>,
          <CIM
            handleSelect={[Function]}
/>,
          <SelectGeographicType
            handleSelect={[Function]}
            measureId={null}
/>,
          <SelectStratificationLevel
            geographicTypeId={null}
            handleSelect={[Function]}
            measureId={null}
/>,
          <GeographicFilter
            geographicTypeId={null}
            handleSelect={[Function]}
            measureId={null}
/>,
          <SelectYears
            handleCheck={[Function]}
            measureId={null}
/>,
          <div
            className="field"
>
            <label
                        className="checkbox"
            >
                        <input
                                    checked={false}
                                    className="input-checkbox"
                                    name="isSmoothed"
                                    onChange={[Function]}
                                    type="checkbox"
                        />
                        Is Smoothed
            </label>
            <p
                        className="help"
            >
                        Note: the majority of measures do not have smoothing values
            </p>
</div>,
          <div
            className="field"
>
            <label
                        className="checkbox"
            >
                        <input
                                    checked={false}
                                    className="input-checkbox"
                                    name="getFullCoreHolder"
                                    onChange={[Function]}
                                    type="checkbox"
                        />
                        getFullCoreHolder
            </label>
</div>,
          <hr />,
          <Preview
            url="https://ephtracking.cdc.gov/apigateway/api/v1/getCoreHolder/null/null/ALL/ALL//0/0"
            validUrl={false}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Retrieving all Values for a Measure",
            "className": "title",
          },
          "ref": null,
          "rendered": "Retrieving all Values for a Measure",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Usage",
            "className": "title is-5",
          },
          "ref": null,
          "rendered": "Usage",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              "https://ephtracking.cdc.gov/apigateway/api/",
              "{",
              "version",
              "}",
              "/getCoreHolder/",
              "{",
              "measureId",
              "}",
              "/",
              "{",
              "stratificationLevelId",
              "}",
              "/",
              "{",
              "geographicTypeIdFilter",
              "}",
              "/",
              "{",
              "geographicItemsFilter",
              "}",
              "/",
              "{",
              "temporal",
              "}",
              "/",
              "{",
              "isSmoothed",
              "}",
              "/",
              "{",
              "getFullCoreHolder",
              "}",
              "[?apiToken][?Variables...]",
            ],
          },
          "ref": null,
          "rendered": Array [
            "https://ephtracking.cdc.gov/apigateway/api/",
            "{",
            "version",
            "}",
            "/getCoreHolder/",
            "{",
            "measureId",
            "}",
            "/",
            "{",
            "stratificationLevelId",
            "}",
            "/",
            "{",
            "geographicTypeIdFilter",
            "}",
            "/",
            "{",
            "geographicItemsFilter",
            "}",
            "/",
            "{",
            "temporal",
            "}",
            "/",
            "{",
            "isSmoothed",
            "}",
            "/",
            "{",
            "getFullCoreHolder",
            "}",
            "[?apiToken][?Variables...]",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Set parameters",
            "className": "title is-5",
          },
          "ref": null,
          "rendered": "Set parameters",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "handleSelect": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "handleSelect": [Function],
            "measureId": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "geographicTypeId": null,
            "handleSelect": [Function],
            "measureId": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "geographicTypeId": null,
            "handleSelect": [Function],
            "measureId": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "handleCheck": [Function],
            "measureId": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <label
                className="checkbox"
>
                <input
                                checked={false}
                                className="input-checkbox"
                                name="isSmoothed"
                                onChange={[Function]}
                                type="checkbox"
                />
                Is Smoothed
</label>,
              <p
                className="help"
>
                Note: the majority of measures do not have smoothing values
</p>,
            ],
            "className": "field",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <input
                    checked={false}
                    className="input-checkbox"
                    name="isSmoothed"
                    onChange={[Function]}
                    type="checkbox"
/>,
                  "Is Smoothed",
                ],
                "className": "checkbox",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "checked": false,
                    "className": "input-checkbox",
                    "name": "isSmoothed",
                    "onChange": [Function],
                    "type": "checkbox",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "input",
                },
                "Is Smoothed",
              ],
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Note: the majority of measures do not have smoothing values",
                "className": "help",
              },
              "ref": null,
              "rendered": "Note: the majority of measures do not have smoothing values",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <label
              className="checkbox"
>
              <input
                            checked={false}
                            className="input-checkbox"
                            name="getFullCoreHolder"
                            onChange={[Function]}
                            type="checkbox"
              />
              getFullCoreHolder
</label>,
            "className": "field",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <input
                  checked={false}
                  className="input-checkbox"
                  name="getFullCoreHolder"
                  onChange={[Function]}
                  type="checkbox"
/>,
                "getFullCoreHolder",
              ],
              "className": "checkbox",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "checked": false,
                  "className": "input-checkbox",
                  "name": "getFullCoreHolder",
                  "onChange": [Function],
                  "type": "checkbox",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
              "getFullCoreHolder",
            ],
            "type": "label",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "url": "https://ephtracking.cdc.gov/apigateway/api/v1/getCoreHolder/null/null/ALL/ALL//0/0",
            "validUrl": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
